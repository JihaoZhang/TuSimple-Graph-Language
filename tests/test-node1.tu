int main()
{	

	node @{int} n1;
	node @{int} n2;

	new n1;
	new n2;

	print (n1.value());
	prints(n1.name());

	n1.setvalue(3);
	n2.setvalue(1);
	print (n1.value());
	prints(n1.name());
	print (n2.value());
	prints(n2.name());

	return 0;
}