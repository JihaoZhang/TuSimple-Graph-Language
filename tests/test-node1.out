0
n1
3
n1
1
n2